api/design/resource.go

```diff
@@ -73,13 +71,11 @@ var _ = Service("resource", func() {
 		Description("List all resources sorted by rating and name")
 		Payload(func() {
 			Attribute("limit", UInt, "Maximum number of resources to be returned", func() {
-				Default(100)
+				Default(1000)
 				Example("limit", 100)
 			})
 		})
-		Result(CollectionOf(Resource), func() {
-			View("withoutVersion")
-		})
+		Result(ResourceResult)
```

Comment:
Can't this be done like this
```
Result(func() {
     Attribute("data", CollectionOf(Resource), "List of resources", func() {
		View("withoutVersion")
	})
})
```

Reply:
yeah it can be this way also.

Reply:
So here we can avoid creating a new type in `type.go`

Reply:
it is used for multiple APIs. and later when we add `metadata`, it has to be added at one place instead of multiple

Reply:
thats why created a new type.

Reply:
Will that be a issue?

Reply:
Nope not any issue :+1: